src/main/kotlin/cn/yiiguxing/plugin/translate/trans/Languages.kt

```diff
@@ -358,18 +359,23 @@ enum class Lang(
     companion object {
         private val cachedValues: List<Lang> by lazy { values().sortedBy { it.langName } }
 
-        val default: Lang = when (Locale.getDefault().language) {
-            Locale.CHINESE.language -> when (Locale.getDefault().country) {
-                "HK", "TW" -> CHINESE_TRADITIONAL
-                else -> CHINESE
+        val default: Lang
+            get() {
+                val dynamicBundleLanguage = DynamicBundle.getLocale().language
```

Comment:
`DynamicBundle.getLocale()` method will cause an exception when the 1.5 version of the Kotlin plug-in is installed:
`java.lang.NoSuchMethodError: 'java.util.Locale com.intellij.DynamicBundle.getLocale()'` (#747)

Do you have any solutions?

Reply:
Kotlin plugin has it's own version of DynamicBundle class for some reason. It should be fixed in next builds of the kotlin plugin (1.5.10+). Since not everyone will update right away, I will add a check similar to #747.